server/models/chat.js

```diff
@@ -0,0 +1,29 @@
+const db = require('../config/db');
+
+exports.create = ({ body, id_to, id_from }) => db.one(
+  `INSERT INTO message (body, sender_id, recipient_id)
+      VALUES ($1, $2, $3)
+      RETURNING message_id`,
+  [body, id_to, id_from]
+);
+
+exports.read = (account_id) => db.any(
+  `SELECT *
+      FROM message
+      WHERE sender_id = $1 OR recipient_id = $1`,
+  [account_id]
+);
+
+exports.update = ({ message_id, body }) => db.none(
+  `UPDATE message
+      SET body          = $2
+      WHERE message_id  = $1`,
+  [message_id, body]
+);
+
+exports.delete = (message_id) => db.none(
```

Comment:
Same as the above, do we need this function? If we are going based on a similar mental model to text messaging, it wouldn't be a big surprise for the user if messages were permanent.

Reply:
Just to clarify, are you saying that text messaging apps don't usually give the user the option to delete messages, or that deleted text messages aren't really "deleted" in the database, but are instead hidden from the sender/recipient?

Reply:
I'm fine not giving the option to delete messages, if that's what you meant.

Reply:
Yeah I mean I think that, at least on iOS individual text messages can't be deleted so maybe we would be wasting our time to give that option.